tests/functional/partial_parsing/test_pp_vars.py

```diff
@@ -1,31 +1,29 @@
-import pytest
```

Comment:
Sorted imports aren't material to test changes, just improving the readability a bit.